### ISSUE TITLE
0.3.5 fails when accessing StartMenuInternet from HKCU

### ISSUE THREAD

**Issue by harrypnyce** [spikespaz/search-deflector#42]
I have encountered an error launching Search Deflector.
The error information follows below.

**File:** `std\windows\registry.d`

**Line:** 510

**Message:**
```
Failed to open requested key: "SOFTWARE\Clients\StartMenuInternet": The system cannot find the file specified. (error 2)
```

**Stack Trace:**
```
0x0105342F
0x010531E2
0x0104B2ED

```

EDIT:  This was **version 0.3.5**

**Comment by harrypnyce** [comment]
I want to apologize in advance, I simply clicked "Yes" to submit a bug report and sent whatever was filled out by default.

Running Windows 10 Pro v1809 on an HP Spectre x360 laptop -- haven't had any issues using this same application on my daily driver (whitebox, desktop PC).  Thanks!

**Comment by spikespaz** [comment]
Well this sucks. I know why this happens though. Use 0.3.4 for now, it shouldn't have this bug. Uninstall the current version from the Control Panel, and download the installer for 0.3.4. Run the installer, deselect the "Automatic Updater" checkbox on the Components page. Go through the rest with the defaults selected.

I'll try to find a fix for the bug tomorrow.

**Comment by spikespaz** [comment]
Would you mind testing this development build? You'll have to uninstall 0.3.4 first. If you install this, make sure "Automatic Updater" is **enabled** this time.

[SearchDeflector-Installer.zip](https://github.com/spikespaz/search-deflector/files/2588052/SearchDeflector-Installer.zip)

Thanks.

**Comment by spikespaz** [comment]
@buffondavid @harrypnyce Did either of you try the version in the message above? I can't release the fix until I know it works.

**Comment by buffondavid** [comment]
@spikespaz Version 0.3.6 works perfectly. Thanks man

**Comment by harrypnyce** [comment]
I'm running v1809 will that matter?  Just downloaded it now, will report back ASAP.  Sorry I missed yer note earlier.

**Comment by spikespaz** [comment]
I have released [0.3.6](https://github.com/spikespaz/search-deflector/releases/tag/0.3.6). The update will automatically pulled to your machine y the existing version.

If you are using the MS Store version, you will have to wait about 3 days for Microsoft to review the changes.

**Comment by harrypnyce** [comment]
Thanks, update to 0.3.6 is working wonderfully.  JUST made the jump from using Chrome as my default browser to Brave.  Less than ~24 hours into it and I'm already loving the adjustment.  Hoping it continues to work well with Search Deflector.  Thanks for the awesome search utility!
